### ISSUE TITLE
Add ability to modify browser permissions for password entries

### ISSUE THREAD

**Issue by 2martens** [keepassxreboot/keepassxc#2385]
When visiting a website with a corresponding password entry, I have to allow access for the browser plugin to be able to use this entry. However if I click on "Remember this decision" and click on the wrong button, there is currently no way to reset the state of the permission. If I clicked on "Deny" it is impossible to retroactively allow access.

## Expected Behavior
I can reset the remembered decision for a particular website so that the popup is appearing again.

## Current Behavior
After the initial popup and clicking on "Remember this decision" it is impossible to reset it.

## Possible Solution
Since the configuration for these browser permission is stored in the database (inaccessible to simple file manipulation), it should be possible to for example right-click on an entry and select a menu option "Reset browser permission". 

## Debug Info
KeePassXC - 2.3.4
Revision: 6fe821c

Libraries:
- Qt 5.5.1
- libgcrypt 1.8.1

Operating system: Linux Mint 18.3
CPU architecture: x86_64
Kernel: linux 4.13.0-39-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey

**Comment by droidmonkey** [comment]
Just go to the entry in question, open it (double click) and go to the advanced page. Find the attribute with a key of "KeePassXC-Browser Settings" or "KeePassHTTP Settings" and remove it. Done.

**Comment by cdrehner** [comment]
For the later readers: The attribute can now be found under 'Properties'.

**Comment by ElenaHerold** [comment]
I have the same issue but don't understand your explanation. Do you mean by entry in question my password entry in the KeePassXC database? But there is no "KeePassXC-Browser Settings" or "KeePassHTTP Settings".
Or are you talking about something in the browser extension?

**Comment by Orchal** [comment]
Hi,

imho, deleting the key whereas we already save a lot of sites preferences is not an acceptable workaround.
Editing the json would be very great!

Thanks

**Comment by domalex** [comment]
@droidmonkey, @cdrehner I finally found the entry and edited it. This way I managed to delete the granted permissions. Thank you for the help.

**Comment by droidmonkey** [comment]
@domalex for future reference, you can easily find entries configured for browser access using the reports tool: https://keepassxc.org/docs/KeePassXC_UserGuide#_browser_statistics
